lib/applications/resolvers.ts

```diff
@@ -264,6 +268,9 @@ export const createNewApplication: Resolver<
           },
         }),
         ...data,
+        postalCode: formatPostalCode(postalCode),
+        shippingPostalCode: shippingPostalCode ? formatPostalCode(shippingPostalCode) : undefined,
+        billingPostalCode: billingPostalCode ? formatPostalCode(billingPostalCode) : undefined,
```

Comment:
```suggestion
        shippingPostalCode: shippingPostalCode && formatPostalCode(shippingPostalCode),
        billingPostalCode: billingPostalCode && formatPostalCode(billingPostalCode),
```
These two fields are required to be `string | null`. If they are null, it means that the form input was that they were null, which carries the intention of removing the postal code data (setting them to null).

Setting these values to `undefined` in the Prisma call will leave them unchanged, which is not desired when we want to remove the entire address (eg. when it's the same as home address)